src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+abstract class Figure implements AreaCalculator {
+
+    public abstract void draw();
+}
```

Comment:
It would be best if you had the interface for the `draw()` method
```suggestion
abstract class Figure implements AreaCalculator, Drawable {
}
```